### PR TITLE
docs: Add changelog of 4.5.4

### PR DESCRIPTION
**Pull request by dengfuping**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 新增 4.5.4 版本的更新日志。          |
| 🇨🇳 Chinese | Add changelog of 4.5.4.          |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered CHANGELOG.en-US.md](https://github.com/ant-design/ant-design/blob/release-4.5.4/CHANGELOG.en-US.md)
[View rendered CHANGELOG.zh-CN.md](https://github.com/ant-design/ant-design/blob/release-4.5.4/CHANGELOG.zh-CN.md)

### CODE REVIEW

File: CHANGELOG.zh-CN.md [hunk 1]
```diff
@@ -15,6 +15,23 @@ timeline: true
...
+
+`2020-08-12`
+
+- 🐞 修复 Form.Item 在 `hidden` 时引用 less 样式时失效的问题。[#26152](https://github.com/ant-design/ant-design/pull/26152)
+- 🐞 修复 Dropdown `overlay` 为 string 时渲染错误的问题。[#26135](https://github.com/ant-design/ant-design/pull/26135)
+- 🐞 修复 Table 删除最后一页的最后一条记录时分页消失的问题。[#26133](https://github.com/ant-design/ant-design/pull/26133) [@QoVoQ](https://github.com/QoVoQ)
+- Tree
+  - 🐞 修复 Tree.DirectoryTree 不能通过 `ref` 调用 `scrollTo` 的问题。[#26129](https://github.com/ant-design/ant-design/pull/26129)
+  - 🐞 修复 Tree 开启 `showLine` 时顶级节点间缺少连接线的问题。[#25991](https://github.com/ant-design/ant-design/pull/25991) [@zhangchen915](https://github.com/zhangchen915)
+- 💄 新增 `@badge-count-background` less 变量。[#26043](https://github.com/ant-design/ant-design/pull/26043) [@sorahn](https://github.com/sorahn)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:28]:
```suggestion
- 💄 新增 `@badge-color` less 变量。[#26043](https://github.com/ant-design/ant-design/pull/26043) [@sorahn](https://github.com/sorahn)
```

https://github.com/ant-design/ant-design/commit/7d500703d7ea565c1c5a0fc365600ef2aa362af2

File: CHANGELOG.zh-CN.md [hunk 2]
```diff
@@ -15,6 +15,23 @@ timeline: true
...
+- RTL
+  - 🐞 修复 Input.Search 在 RTL 模式下的边框样式。[#26156](https://github.com/ant-design/ant-design/pull/26156)
+  - 🐞 修复 DatePicker 日期范围选择中开始时间和结束时间在 RTL 模式下边界弧度的限制问题。[#26149](https://github.com/ant-design/ant-design/pull/26149)
+  - 🐞 修复 DatePicker 时间范围选择中开始时间和结束时间相等时在 RTL 模式下的显示问题。[#26157](https://github.com/ant-design/ant-design/pull/26157)
```
Comment (afc163) [inline CHANGELOG.zh-CN.md:32]: #26158

**Review by zombieJ — APPROVED** [review]

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (afc163) [reply CHANGELOG.zh-CN.md:32]: 这几个可以合并写修复 DatePicker 一系列 RTL 样式问题。

**Comment by afc163** [comment]
#26160

**Review by xrkffgg — APPROVED** [review]

File: CHANGELOG.zh-CN.md (hunk 2 above)
Comment (xrkffgg) [reply CHANGELOG.zh-CN.md:32]: Input 相关的 2 个
DatePicker 相关的 3 个

Comment (dengfuping) [reply CHANGELOG.zh-CN.md]: - [x] #26158 和 #26160 的更新日志已加上；
- [x] DatePicker 在 RTL 模式下样式修复的更新日志已合并；

**Review by afc163 — APPROVED** [review]

File: CHANGELOG.en-US.md [hunk 3]
```diff
@@ -15,6 +15,24 @@ timeline: true
...
+
+`2020-08-12`
+
+- 🐞 Fix Form.Item `hidden` not working in less. [#26152](https://github.com/ant-design/ant-design/pull/26152)
+- 🐞 Fix Dropdown render issue when `overlay` is string. [#26135](https://github.com/ant-design/ant-design/pull/26135)
+- 🐞 Fix Table pagination disappear when last record in last page is deleted. [#26133](https://github.com/ant-design/ant-design/pull/26133) [@QoVoQ](https://github.com/QoVoQ)
+- Tree
+  - 🐞 Fix Tree.DirectoryTree can not use `scrollTo` with ref. [#26129](https://github.com/ant-design/ant-design/pull/26129)
+  - 🐞 Fix Tree lost line between top level nodes when enable `showLine`. [#25991](https://github.com/ant-design/ant-design/pull/25991) [@zhangchen915](https://github.com/zhangchen915)
+- 💄 Add `@badge-count-background` less variable. [#26043](https://github.com/ant-design/ant-design/pull/26043) [@sorahn](https://github.com/sorahn)
```
Comment (xrkffgg) [inline CHANGELOG.en-US.md:28]:
```suggestion
- 💄 Add `@badge-color` less variable. [#26043](https://github.com/ant-design/ant-design/pull/26043) [@sorahn](https://github.com/sorahn)
```

Comment (dengfuping) [reply CHANGELOG.en-US.md:28]: - 看 [代码](https://github.com/ant-design/ant-design/pull/26043/files#diff-4b7c05557620faa918b0919c047d049a) 加的是 `@badge-count-background` 变量？
image

Comment (dengfuping) [reply CHANGELOG.en-US.md:28]: - OK，看到了半小时前更新了变量命名。

File: CHANGELOG.zh-CN.md (hunk 1 above)
Comment (dengfuping) [reply CHANGELOG.zh-CN.md:28]: - 已修正

**Pull request merged**